tests/fuzz/fuzz-bech32.c

```diff
@@ -0,0 +1,49 @@
+#include <stdint.h>
+#include <string.h>
+#include <tests/fuzz/libfuzz.h>
+
+#include <common/bech32.h>
+
+void init(int *argc, char ***argv)
+{
+}
+
+void run(const uint8_t *data, size_t size)
+{
+	const char hrp_inv[5] = "lnbc\0", hrp_addr[3] = "bc\0";
```

Comment:
do these constants not exist elsewhere in the project?

Reply:
Didn't find them